test/lib/lib-helpers/request.test.js

```diff
@@ -36,6 +39,39 @@ test('isExternalPost returns whether req.postType is external', (t) => {
   t.falsy(requestHelper.isExternalPost(t.context.req));
 });
 
+// isKeyword
+test('isKeyword returns whether req.keyword is set', (t) => {
+  t.context.req.keyword = stubs.getKeyword();
+  t.truthy(requestHelper.isKeyword(t.context.req));
+  t.context.req.keyword = null;
+  t.falsy(requestHelper.isKeyword(t.context.req));
+});
+
+// isStartCommand
+test('isStartCommand returns false if messageText undefined', (t) => {
+  sandbox.stub(requestHelper, 'messageText')
+    .returns(null);
+  t.falsy(requestHelper.isStartCommand(t.context.req));
+});
+
+test('isStartCommand returns true if messageText is Start Command with leading and trailing whitespace', (t) => {
+  sandbox.stub(requestHelper, 'messageText')
+    .returns(` ${startCommand} `);
+  t.truthy(requestHelper.isStartCommand(t.context.req));
+});
+
+test('isStartCommand returns true if messageText is camelcase Start Command', (t) => {
+  sandbox.stub(requestHelper, 'messageText')
+    .returns(camelCase(startCommand));
+  t.truthy(requestHelper.isStartCommand(t.context.req));
+});
+
+test('isStartCommand returns true if messageText is lowercase Start Command', (t) => {
+  sandbox.stub(requestHelper, 'messageText')
+    .returns(startCommand.toLowerCase());
+  t.truthy(requestHelper.isStartCommand(t.context.req));
```

Comment:
It will always be START although for our `isStartCommand` should perform a  case-insensitive trimmed match, as users are texting the command back to the SMS chatbot to begin a photo post.